### DISCUSSION TITLE
running python3 using script

### DISCUSSION THREAD

**Discussion by eddmy** [atom/atom#22551] [Show and tell]
After configuring Atom to use it as a python environment (IDE) On MacOS the 'script run' (shortcut cmd-I) will run python2.7
That version of python is needed by the OS and can't be removed.
To make cmd-I run python3 instead replace 'python' with 'python3' in the following file:
.atom/packages/script/lib/grammars/python.js

Two lines should read: 
command: "python3",

**Comment by idleberg** [comment]
Even though this approach is [mentioned](https://github.com/atom-community/atom-script#:~:text=Change%20Default,python.js) in script`s README, it's a bad idea to so. Whenever the package updates, your changes are gone. The correct way is to [use a profile](https://github.com/atom-community/atom-script#usage).
